### ISSUE TITLE
_IncompatibleKeys && constant

### ISSUE THREAD

**Issue by DeoooChen** [SaMMyCHoo/Light-VQA-plus#3]
Thanks for your wonderful work! But I meet a problem below:

**_IncompatibleKeys(missing_keys=['head1.weight', 'head1.bias', 'head2.weight', 'head2.bias', 'CrossAttn_head.linear_A.weight', 'CrossAttn_head.linear_A.bias', 'CrossAttn_head.linear_B.weight', 'CrossAttn_head.linear_B.bias', 'CrossAttn_head.cross_attention.query_linear.weight', 'CrossAttn_head.cross_attention.query_linear.bias', 'CrossAttn_head.cross_attention.key_linear.weight', 'CrossAttn_head.cross_attention.key_linear.bias', 'CrossAttn_head.cross_attention.value_linear.weight', 'CrossAttn_head.cross_attention.value_linear.bias', 'CrossAttn_head.output_linear.weight', 'CrossAttn_head.output_linear.bias'], unexpected_keys=['layers.0.blocks.1.attn_mask', 'layers.1.blocks.1.attn_mask', 'layers.2.blocks.1.attn_mask', 'layers.2.blocks.3.attn_mask', 'layers.2.blocks.5.attn_mask', 'layers.2.blocks.7.attn_mask', 'layers.2.blocks.9.attn_mask', 'layers.2.blocks.11.attn_mask', 'layers.2.blocks.13.attn_mask', 'layers.2.blocks.15.attn_mask', 'layers.2.blocks.17.attn_mask'])**
Trainable params: 58.70 million
Starting training:
Epoch 1 averaged training loss: 15.8626
The current learning rate is 0.000250
D:\anaconda\envs\kk\lib\site-packages\scipy\optimize\_minpack_py.py:906: OptimizeWarning: Covariance of the parameters could not be estimated
  warnings.warn('Covariance of the parameters could not be estimated',
D:\anaconda\envs\kk\lib\site-packages\scipy\stats\_stats_py.py:4424: ConstantInputWarning: An input array is **constant**; the correlation coefficient is not defined.
  warnings.warn(stats.ConstantInputWarning(msg))
D:\anaconda\envs\kk\lib\site-packages\scipy\stats\_stats_py.py:4916: ConstantInputWarning: An input array is **constant**; the correlation coefficient is not defined.
  warnings.warn(stats.ConstantInputWarning(warn_msg))
Epoch 1 completed. The result on the test databaset: **SRCC: nan, KRCC: nan, PLCC: nan**, and RMSE: 1.0399

BTW, I actually download the weights and put it into the right path:
    parser.add_argument('--weights', type=str, default='swin_small_patch4_window7_224.pth', help='initial weights path')

**Comment by SaMMyCHoo** [comment]
Thanks for your attention into our work. The “_IncompatibleKeys” warning is expected, just ignore it. As for the nan problem, it may have something to do with the parameters initialization. You may try different initialization strategies and see it it works.
